packages/components/src/components/molecules/Filters/Filters.tsx

```diff
@@ -55,13 +56,19 @@ function getActiveFilterLabel(
 ): string {
   const options =
     filter?.options ?? fieldOptions.get(filter?.field.property['@id']) ?? []
+
   if (filter.id.endsWith('[between]')) {
     value = (value as (string | number)[]).join('-')
   }
+
   let label = `${filter?.label}: ${value}`
-  const option = options.find((option) => option.value === value)
+
+  const option: IOption<unknown> = options.find((option) => {
+    return option.value === value
+  })
+
   if (option) {
-    label = `${filter?.label}: ${option.label}`
+    return (label = `${filter?.label}: ${option.label}`)
```

Comment:
L'affectation à la variable `label` me semble pas utile (et du coup perturbante):
```suggestion
    return `${filter?.label}: ${option.label}`
```